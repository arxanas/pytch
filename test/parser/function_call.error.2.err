EXPECTED_END_OF_ARGUMENT_LIST[1018] in test/parser/function_call.error.2.pytch, line 3, character 6:
Error: I was expecting a ',' or ')' after the previous argument.
   +-------------------------------------------------------------------------+
   | test/parser/function_call.error.2.pytch                                 |
 2 |   bar                                                                   |
 3 |   baz qux)                                                              |
   |      ^ Error: I was expecting a ',' or ')' after the previous argument. |
   +-------------------------------------------------------------------------+
