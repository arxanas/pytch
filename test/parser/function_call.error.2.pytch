let foo = foo(
  bar
  baz qux)
