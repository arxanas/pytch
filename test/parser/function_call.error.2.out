SyntaxTree
    LetExpr
        Token 'let'
        VariablePattern
            Leading ' '
            Token 'foo'
        <missing>
        Leading ' '
        Token '='
        FunctionCallExpr
            IdentifierExpr
                Leading ' '
                Token 'foo'
            ArgumentList
                Token '('
                Trailing '\n'
                Argument
                    BinaryExpr
                        IdentifierExpr
                            Leading '  '
                            Token 'bar'
                            Trailing '\n'
                        Token DUMMY_SEMICOLON ''
                        IdentifierExpr
                            Leading '  '
                            Token 'baz'
                    <missing>
                Leading ' qux'
                Token ')'
                Trailing '\n'
        Token DUMMY_IN ''
        <missing>
    Token EOF ''
